Gemfile

```diff
@@ -12,6 +12,8 @@ gem 'devise'
 gem 'httparty'
 gem 'nokogiri'
 gem 'pg', '~> 0.18'
+gem 'pghero'
+gem 'pg_query', '>= 0.9.0'
```

Comment:
Gems should be sorted in an alphabetical order within their section of the Gemfile. Gem `pg_query` should appear before `pghero`.